tests/core/conftest.py

```diff
@@ -334,8 +334,10 @@ def wait_for_condition(condition_type, status, client, obj, timeout=45):
         obj = client.reload(obj)
         delta = time.time() - start
         if delta > timeout:
-            msg = 'Timeout waiting for [{}:{}] for condition after {}' \
-                ' seconds'.format(obj.type, obj.id, delta)
+            msg = 'Expected condition {} to have status {}'\
+                'Timeout waiting for [{}:{}] for condition after {} ' \
+                'seconds\n {}'.format(condition_type, status, obj.type, obj.id,
+                                      delta, str(obj).replace(",", ",\n"))
```

Comment:
Drop the replace, this makes the output huge on something like a cluster.

Reply:
```
raise Exception(msg)
E               Exception: Expected condition Downloaded to have status UnknownTimeout waiting for [kontainerDriver:kd-f6jgl] for condition after 46.906996965408325 seconds
E                {'actions': {'deactivate': 'https://10.20.0.17:8443/v3/kontainerDrivers/kd-f6jgl?action=deactivate'}, 'active': True, 'actualUrl': 'https://github.com/rancher/kontainer-engine-driver-example/releases/download/v0.2.1/kontainer-engine-driver-example-darwin', 'annotations': {'lifecycle.cattle.io/create.mgmt-kontainer-driver-lifecycle': 'true'}, 'baseType': 'kontainerDriver', 'builtIn': False, 'checksum': '', 'conditions': [{'lastUpdateTime': '2019-03-11T14:23:41-07:00', 'status': 'True', 'type': 'Downloaded'}, {'lastUpdateTime': '2019-03-11T14:23:41-07:00', 'status': 'True', 'type': 'Installed'}, {'lastUpdateTime': '2019-03-11T14:23:46-07:00', 'status': 'True', 'type': 'Active'}], 'created': '2019-03-11T21:23:37Z', 'createdTS': 1552339417000, 'creatorId': 'user-kj5pm', 'executablePath': 'management-state/bin/kontainer-engine-driver-example', 'id': 'kd-f6jgl', 'labels': {'cattle.io/creator': 'norman'}, 'links': {'remove': 'https://10.20.0.17:8443/v3/kontainerDrivers/kd-f6jgl', 'self': 'https://10.20.0.17:8443/v3/kontainerDrivers/kd-f6jgl', 'update': 'https://10.20.0.17:8443/v3/kontainerDrivers/kd-f6jgl'}, 'name': 'example', 'state': 'active', 'transitioning': 'no', 'transitioningMessage': '', 'type': 'kontainerDriver', 'uiUrl': '', 'url': 'https://github.com/rancher/kontainer-engine-driver-example/releases/download/v0.2.1/kontainer-engine-driver-example-darwin', 'uuid': 'eeed239d-4443-11e9-9673-025000000001'}
```
This is now how it will appear in logs